pages/building-and-deploying.md

```diff
@@ -60,6 +60,41 @@ npm run serve
 ### Deploying `prpl-server`
 After building, the contents of `server/` contains all the files and configuration necessary to run the app in production. The provided `server/package.json` specifies server dependencies and the start command which can be used on almost any hosting service that supports Node.js.
 
+### Using `prpl-server` as an express module
```

Comment:
"prpl-server as a library" should be its own top section between "prpl-server (recommended)" and "Static hosting". Right now the below App Engine and Firebase steps don't follow from this section.

Table of contents needs to be updated.

Reply:
OK will get on to it tomorrow, going to sleep right now...

Reply:
Sorry I am late. Scheduled to work on it tomorrow.

Reply:
(...actually done now. The joy of crazy Saturday nights when you are a parent!)